android/.gradle/buildOutputCleanup/cache.properties

```diff
@@ -1,2 +1,2 @@
 #Tue Jul 09 18:11:55 IST 2019
-gradle.version=4.10.2
```

Comment:
check this gradle folder is necessary, since it generated automatically when we build

Reply:
Since it's automatically generated and not needed. I'll remove this